tests/test_threads.py

```diff
@@ -180,3 +181,16 @@ def test_should_abort():
 
         should_abort()
         ThreadPool().query_abort.called_once_with()
+
+
+def test_stop_threads():
```

Comment:
This should be a method of the class `TestThreadPool`, which bundles the tests related to methods of `ThreadPool`.